javascript/javascript1/week1/smart-ease io/Housey-pricey.js

```diff
@@ -0,0 +1,27 @@
+const PeterHouseWidth = 8
+const PeterHouseDepth = 10
+const PeterHouseHeight = 10
+const PeterGardenSizeInM2 = 100
+const PeterHouseCost = 2500000
+const PeterVolumeInMeters = PeterHouseWidth * PeterHouseDepth * PeterHouseHeight
+PeterHousePrice = PeterVolumeInMeters * 2.5 * 1000 + PeterGardenSizeInM2 * 300;
+
+if (PeterHouseCost < PeterHousePrice) {
+    console.log(`Peter paid too little`)
+} else {
+    console.log(`Peter paid too much`)
+}
+
+const JuliaHouseWidth = 5
+const JuliaHouseDepth = 11
+const JuliaHouseHeight = 8
+const JuliaGardenSizeInM2 = 70
+const JuliaHouseCost = 1000000
+const JuliaVolumeInMeters = JuliaHouseWidth * JuliaHouseDepth * JuliaHouseHeight
+JuliaHousePrice = JuliaVolumeInMeters * 2.5 * 1000 + JuliaGardenSizeInM2 * 300;
```

Comment:
declare variable with 'var/let/const' etc. Please check google , how it is different from if we declare without keys'